Fintech Framework/src/driver/repository/inmemory/AccountInMemoryRepository.java

```diff
@@ -15,26 +20,41 @@ public static AccountInMemoryRepository getInstance() {
         return AccountInMemoryRepository.instance;
     }
 
-    private AccountInMemoryRepository() {
+    private static final String FILE_PATH = "Fintech Framework/src/database/accounts.json";
```

Comment:
This file path is hard code. It should be relative and don't affect by environment.